docs/src/prototypes/popups/GridImagePicker/GridImagePickerItem.tsx

```diff
@@ -0,0 +1,46 @@
+import { Image, Button } from '@stardust-ui/react'
+
+import * as React from 'react'
+import * as _ from 'lodash'
+
+export interface GridPickerItemProps {
+  as?: keyof React.ReactHTML
+  title?: string
+  imageSrc: string
+  onClick?: (e: React.SyntheticEvent, props: GridPickerItemProps) => void
+}
+
+const imageStyles = {
+  width: '100%',
+}
+
+const imageButtonStyles = {
+  minWidth: '56px',
+  height: '56px',
+  padding: '0',
+  background: '#fff',
+}
+
+class GridImagePickerItem extends React.Component<GridPickerItemProps> {
+  static defaultProps = {
+    as: 'li',
+  }
+
+  handleClick = e => {
+    _.invoke(this.props, 'onClick', e, this.props)
+  }
+
+  render() {
+    const { title, imageSrc, as: ElementType } = this.props
+
+    return (
+      <ElementType>
+        <Button styles={imageButtonStyles} onClick={this.handleClick} title={title} role="listitem">
```

Comment:
const { onClick } = this.props
Then on the Button, you can just use onClick={onClick}
No need for additional handler if you are just invoking the onClick from the props there.